spec/MultiDomainInventoryProxy.yaml

```diff
@@ -9333,27 +9333,27 @@ paths:
                       minLength: 3
                       description: >
                         'If body provided, name of RegistryOffice application
-                        find [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-0-http-c-ro-2-0-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
+                        find [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-c-ro-2-1-2000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
```

Comment:
mdip-1-0-2-http-c-ro-2-1-2000 to be corrected to mdip-1-0-2-http-c-ro-2-1-2-000. 
Similarly , all the following instance in the specification file needs the same correction, 
**RegistryOffice :**
mdip-1-0-2-op-c-bm-ro-2-1-2000, 
mdip-1-0-2-http-c-ro-2-1-2000,
 mdip-1-0-2-tcp-c-ro-2-1-2000
**ExecutionAndTraceLog:**
mdip-1-0-2-http-c-eatl-2-1-2000, 
mdip-1-0-2-op-c-bs-eatl-2-1-2000, 
mdip-1-0-2-tcp-c-eatl-2-1-2000
**Oamlog:**
mdip-1-0-2-http-c-ol-2-1-2000, 
mdip-1-0-2-op-c-bs-ol-2-1-2000, 
mdip-1-0-2-tcp-c-ol-2-1-2000
**AccessAdministrator:**
mdip-1-0-2-http-c-aa-2-1-2000, 
mdip-1-0-2-op-c-bs-aa-2-1-2000, 
mdip-1-0-2-tcp-c-aa-2-1-2000
**ApplicationLayerTopology:**
mdip-1-0-2-http-c-alt-2-1-2000, 
mdip-1-0-2-op-c-bm-alt-2-1-2001, 
mdip-1-0-2-op-c-bm-alt-2-1-2002, 
mdip-1-0-2-op-c-bm-alt-2-1-2003, 
mdip-1-0-2-op-c-bm-alt-2-1-2004, 
mdip-1-0-2-op-c-bm-alt-2-1-2005, 
mdip-1-0-2-tcp-c-alt-2-1-2000

Reply:
just noticed that as well, will fix in all specs